test_framework/scripts/rancher/terraform/main.tf

```diff
@@ -0,0 +1,65 @@
+terraform {
+  required_providers {
+    rancher2 = {
+      source  = "rancher/rancher2"
+      version = "~> 1.24"
+    }
+  }
+}
+
+provider "rancher2" {
+  api_url   = var.api_url
+  insecure  = true
+  access_key = var.access_key
+  secret_key = var.secret_key
+}
+
+resource "rancher2_catalog_v2" "longhorn_repo" {
+  cluster_id = "local"
+  name = "longhorn-repo"
+  git_repo = var.longhorn_git_repo
+  git_branch = var.longhorn_git_branch
+}
+
+resource "rancher2_app_v2" "longhorn_app" {
+  cluster_id = "local"
+  name = "longhorn-app"
+  namespace = "longhorn-system"
+  repo_name = "longhorn-repo"
+  chart_name = "longhorn"
+  #chart_version = "100.1.2+up1.2.4" # if chart_version is not specified, the latest version would be installed
```

Comment:
This is good. Probably we should keep this, then we can test different versions if needed in other cases.

Reply:
Modified.

Reply:
👍